app/main/views.py

```diff
@@ -10,6 +10,11 @@ def index():
     return render_template('main/index.html')
 
 
+@main.route('/map', methods=['GET', 'POST'])
+def mapusers():
+    return render_template('main/mapview.html', users=User.query.all())
```

Comment:
let's change the name to something besides `mapview.html`. We don't need to say `view` because it's an html page, and also there should be an underscore separating multiword file names.